### DISCUSSION TITLE
When creating an issue from a task list, I want to be able to use our existing issue templates

### DISCUSSION THREAD

**Discussion by 0xdevalias** [community/community#4287] [Projects and Issues]
I haven't deeply played with this feature yet, but from the quick look I just had, it seemed to create a blank issue when I clicked on the 'convert to issue' rather than letting me choose one of our existing issue templates. This makes the feature all but unusable for us, as we use our issue templates to provide a consistent layout + containing all of the needed fields.

**Comment by akinaito** [comment]
👍  This seems to be the case. We'd like to use our project board as single source for prioritized tasks, but are not able to apply any structure to those issues that get converted from the project board since it just opens a blank issue

Also mentioned [here](https://github.community/t/where-do-i-post-feature-requests-for-github/1501/13?u=akinaito) by @lbphillipvose

**Reply by r002** [reply]
I also have one bit of feedback related to this.  In our project, we use one `repo` strictly as an `Issues repo` (that outside users/anyone can submit to) and then we have another repo that is our dedicated `dev repo` (issues specifically related to dev work).  In other projects, I've seen this multi-repo setup as well (eg. [uBlockOrigin](https://github.com/uBlockOrigin/uBlock-issues) has a community-maintained issues repo too that's distinct from its dev repo); bc of how permissions work, I believe this is a common setup for many projects?

This might be a bridge too far in terms of overengineering the `GitHub Issues Writer GUI`, but it'd be nice if when I clicked on the `"Convert to issue"` button next to a task that I got an option somewhere to decide _**which repo**_ to create the new issue in.

**Reply by kevprice** [reply]
It now appears to do the above (ask which repo on 'Convert issue') , which is brilliant. However it doesn't use the default issue template for that repo, it still creates a blank issue.
It means the only way to do it is to go to Issues and click 'create new' to get the default template rather than using the projects GUI

**Comment by umutseven92** [comment]
This would be a great addition. 
For now, what we do is crate the sub-issue first, then add it to the parent issue via `- [ ] #ISSUE_NUMBER`. This accomplishes the same thing.

**Comment by filipe-j-delgado-alb** [comment]
I vote for this improvement, it would help me a lot.

**Comment by shieldgenerator7** [comment]
I have to say that creating multiple issues is currently a hassle no matter which way you do it.
Add an item to a project board? It takes an extra 3 clicks per issue!
Add an issue using the issue pages? It takes only 1 click to make the issue, but an extra 3 clicks per issue to get them on a board!

All these extra clicks really add up and create friction that makes the projects hard to use.

It would be great if there was a setting to
(A) Make a project auto-make issues for a specific project (so we dont have to keep selecting the same one all the time)
(B) Auto-make items into issues (so we dont have to manually turn each item into an issue)

Those two changes would make my life so much easier while using GitHub projects
